src/net.py

```diff
@@ -125,12 +129,14 @@ def __init__(self, name, dimensions, input_var, y, config,
             [i for i in [self.input_var] if i],
             lasagne.layers.get_output(self.network, deterministic=True))
         self.record = None
+
         try:
             self.timestamp
         except AttributeError:
             self.timestamp = get_timestamp()
         self.minibatch_iteration = 0
 
+
```

Comment:
Just a general comment but I don't think there need to be two spaces between functions that belong to a class. I think that applies to only functions outside of a class.